unit/suites/test_mesh.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import sys
+import unittest
+import warnings
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+
+@unittest.skipIf(sys.platform.startswith("win"), 'Mesh tests on windows platform not supported')
+class TestSuite_Mesh(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        print(' MESH '.center(70, '='), file=sys.stderr)
+        print('-' * 70, file=sys.stderr)
+        
+        # Start first server
+        self.srv = TarantoolServer()
+        self.srv.script = 'unit/suites/box.lua'
+        self.srv.start()
+        self.srv.admin("box.schema.user.create('test', { password = 'test', if_not_exists = true })")
+        self.srv.admin("box.schema.user.grant('test', 'execute', 'universe')")
+        
+        # Start second server
+        self.srv2 = TarantoolServer()
+        self.srv2.script = 'unit/suites/box.lua'
+        self.srv2.start()
+        self.srv2.admin("box.schema.user.create('test', { password = 'test', if_not_exists = true })")
+        self.srv2.admin("box.schema.user.grant('test', 'execute', 'universe')")
+
+        # get_nodes function contains both servers' addresses
+        get_nodes = " \
+            function get_cluster_nodes() \
+                return { '%s:%d', '%s:%d' } \
+            end" % (self.srv.host, self.srv.args['primary'], self.srv2.host, self.srv2.args['primary'])
+
+        # Create get_nodes function on servers
+        self.srv.admin(get_nodes)
+        self.srv2.admin(get_nodes)
+
+        # Create srv_id function (for testing purposes)
+        self.srv.admin("function srv_id() return 1 end")
+        self.srv2.admin("function srv_id() return 2 end")
+
+
+    def setUp(self):
+        # prevent a remote tarantool from clean our session
+        if self.srv.is_started():
+            self.srv.touch_lock()
```

Comment:
It does not run on windows, not?

Reply:
Not working on win. May be some fixes need to be done in test-run repo

Reply:
So maybe skip setup / cleanup too on Windows?

Reply:
Didn't helped